kratos/utilities/parallel_levelset_distance_calculator.h

```diff
@@ -603,9 +603,12 @@ class ParallelDistanceCalculator
                 if (b > 0) q = -0.5 * (b + sqrt_det);
                 else q = -0.5 * (b - sqrt_det);
                 root1 = q / a;
-                root2 = c / q;
-                if (root1 > root2) distance = root1;
-                else distance = root2;
+                distance = root1;
+                if( q != 0.0)
+                {
+                    root2 = c / q;
+                    if (root2 > root1) distance = root2;
+                }
```

Comment:
```suggestion
                if(std::abs(q) > 0.0) {
                    root2 = c / q;
                    if (root2 > root1) distance = root2;
                }
```
To avoid checking against zero.

Reply:
Thanks for your suggestion, i committed it.